### PR TITLE
修复评论管理后台表情包过大的问题

### PR DESCRIPTION
**Pull request by oCoke**
Issue: #295

### CODE REVIEW

**Comment by Mister-Hope** [comment]
Please add it in custom.scss

**Comment by oCoke** [comment]
OK.

File: packages/admin/src/style/style.css [hunk 1]
```diff
@@ -2566,3 +2566,4 @@ div.token-input-dropdown ul li.token-input-selected-dropdown-item {
 .invisible {
   visibility: hidden;
 }
+
```
Comment (Mister-Hope) [inline packages/admin/src/style/style.css:2569]: Please remove this extra line otherwise the linter will fail.

**Review by Mister-Hope — CHANGES_REQUESTED** [review]

File: packages/admin/src/style/custom.scss [hunk 2]
```diff
@@ -33,3 +33,10 @@ div[tabindex='-1'] {
 .account-item.github.bind path {
   fill: #1b1f23;
 }
+
+.vemoji { 
+  display: inline-block;
+  vertical-align: middle;
+  width: 1.3rem;
+  margin: .25rem;
```
Comment (Mister-Hope) [inline packages/admin/src/style/custom.scss:41]: This should be 0.25rem to pass prettier

**Review by Mister-Hope — CHANGES_REQUESTED** [review]

File: packages/admin/src/style/custom.scss [hunk 3]
```diff
@@ -33,3 +33,10 @@ div[tabindex='-1'] {
 .account-item.github.bind path {
   fill: #1b1f23;
 }
+
+.vemoji {
```
Comment (Mister-Hope) [inline packages/admin/src/style/custom.scss:37]:
```suggestion
.vemoji {
```

**Review by Mister-Hope — CHANGES_REQUESTED** [review]

**Pull request merged**